e2e/utils/cardActions/scroll.js

```diff
@@ -0,0 +1,8 @@
+module.exports.scrollCardBy = async(browser, cardSelector, scrollByY)=>{
```

Comment:
you'll have to send the scrollByY to the executeAsync function too, and use that value.
10, 10 are just placeholders?